support-frontend/nginx/README.md

```diff
@@ -1,19 +1,15 @@
 # Support™ NGINX
+#### Installing NGINX
 
-- Install dependencies:
-```bash
-brew tap "guardian/devtools"
-brew install "guardian/devtools/dev-nginx"
-```
+- nginx will be installed and configured by running [setup.sh](setup.sh) script from the root of the `support-frontend` project:
 
-Or from the `nginx` directory:
-```bash
-brew bundle
 ```
-- Run [setup.sh](setup.sh) script from the root of the `support-frontend` project:
+./setup.sh
+```
 
+#### Running nginx
 ```
-./nginx/setup.sh
+sudo nginx
```

Comment:
Ii don't run this with sudo - not sure if it depends on the way one installs it

Reply:
could flag it as optional?

Reply:
I thought it had to be run with `sudo` in order to bind to port 80?

Reply:
actually I've just tried it (I had to `sudo chown joseph_smith:admin /usr/local/var/log/nginx/access.log` and `sudo chown joseph_smith:admin /usr/local/var/log/nginx/error.log`), and it can indeed run without `sudo`!

Reply:
interesting - I always thought you had to be running as `root` to bind to port below 1024. https://unix.stackexchange.com/questions/10735/allowing-a-user-to-let-listen-to-a-port-below-1024

maybe it applies only to Linux to Mac OS X? Maybe @akash1810 can shed some light since he's our current nginxpert...

Reply:
my understanding was similar to yours, binding to a low port required `sudo`.